### ISSUE TITLE
Questions about loss3

### ISSUE THREAD

**Issue by zhang2002forwin** [mkshing/ziplora-pytorch#32]
The training process described in the paper is as follows, but when I reproduced it, I found that the value of loss3 changed very little. What could be the problem?
Here is my code:
` accelerate launch train_dreambooth_ziplora_sdxl.py \
            --pretrained_model_name_or_path="/media/stable-diffusion-xl-base-1.0" \
            --output_dir="ziplora_lora/${cnt_base_name}-${sty_base_name}/" \
            --lora_name_or_path="$cnt_dir/pytorch_lora_weights.safetensors" \
            --instance_prompt="the [ccc]" \
            --instance_data_dir="/media/content_image/$cnt_base_name" \
            --lora_name_or_path_2="$sty_dir/pytorch_lora_weights.safetensors" \
            --instance_prompt_2="in [sss] style" \
            --instance_data_dir_2="/media/Styles/$sty_base_name" \
            --resolution=1024 \
            --train_batch_size=1 \
            --learning_rate=5e-5 \
            --similarity_lambda=0.01 \
            --lr_scheduler="constant" \
            --lr_warmup_steps=0 \
            --max_train_steps=100 \
            --validation_prompt="the [ccc], in [sss] style" \
            --validation_epochs=2000 \
            --seed="0" \
            --mixed_precision="fp16" \
            --gradient_checkpointing \
            --use_8bit_adam `
<img width="860" height="122" alt="Image" src="https://github.com/user-attachments/assets/b21791f3-c5ad-45b7-ab4b-483f235322c2" />

<img width="704" height="61" alt="Image" src="https://github.com/user-attachments/assets/7fdcbadd-2db5-405c-9616-a261a891a8d0" />
